release.md

```diff
@@ -6,8 +6,10 @@ Before starting the release process, verify the following:
 
 - All work required for this release has been completed and the team is ready to release.
 - [All Github Actions Tests are green on main for Woodwork](https://github.com/alteryx/woodwork/actions?query=branch%3Amain+workflow%3ATests).
-- [Featuretools Tests are green with Woodwork main](https://github.com/alteryx/featuretools/actions/workflows/unit_tests_with_woodwork_main_branch.yml?query=branch%3Amain)
-- [EvalML Tests are green with Woodwork main](https://github.com/alteryx/evalml/actions/workflows/unit_tests_with_woodwork_main_branch.yml?query=branch%3Amain)
+- Featuretools Tests are green with Woodwork main:
+  - [![Unit Tests - Featuretools with Woodwork main branch](https://github.com/alteryx/featuretools/actions/workflows/unit_tests_with_woodwork_main_branch.yml/badge.svg?branch=main)](https://github.com/alteryx/featuretools/actions/workflows/unit_tests_with_woodwork_main_branch.yml)
+- EvalML Tests are green with Woodwork main:
+  - [![Unit Tests - EvalML with Featuretools main branch](https://github.com/alteryx/evalml/actions/workflows/unit_tests_with_featuretools_main_branch.yml/badge.svg?branch=main)](https://github.com/alteryx/evalml/actions/workflows/unit_tests_with_featuretools_main_branch.yml)
```

Comment:
The badge for this is showing the results of EvalML with Featuretools main, not the results of EvalML with Woodwork main, which I think is what we want.

![image](https://user-images.githubusercontent.com/4307001/168160959-8ed30cf8-34f6-4df2-aa80-3872b7b56f5f.png)

Reply:
Fixed